bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | tail -n 1)
 
+ADAM_JARS=$(echo "$ADAM_JARS" | tail -n +1 | perl -pe 's/\n/,/ unless eof')
```

Comment:
What if we have `compute-adam-jars.sh` take one cmdline flag, that tells it to strip the CLI jar or not, and then have it do the `perl -pe 's/\n/,/ unless eof'`, since that's what both callers want?

Then `compute-adam-jars.sh` could do some more structured things to find and remove the CLI jar, and we'd expose a clean interface to that to the two caller scripts.